superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -614,214 +579,145 @@ class SqlEditor extends React.PureComponent {
+              </Button>
             )}
-          />
-        ) : (
-          this.queryPane()
-        )}
-        <StyledModal
-          visible={this.state.showCreateAsModal}
-          title={t(createViewModalTitle)}
-          onHide={() => {
-            this.setState({ showCreateAsModal: false });
-          }}
-          footer={
-            <>
+            {createAs === CtasEnum.VIEW && (
               <Button
-                onClick={() => this.setState({ showCreateAsModal: false })}
+                buttonStyle="primary"
+                disabled={ctas.length === 0}
+                onClick={createViewAs}
               >
-                Cancel
+                Create
               </Button>
-              {this.state.createAs === CtasEnum.TABLE && (
-                <Button
-                  buttonStyle="primary"
-                  disabled={this.state.ctas.length === 0}
-                  onClick={this.createTableAs.bind(this)}
-                >
-                  Create
-                </Button>
-              )}
-              {this.state.createAs === CtasEnum.VIEW && (
-                <Button
-                  buttonStyle="primary"
-                  disabled={this.state.ctas.length === 0}
-                  onClick={this.createViewAs.bind(this)}
-                >
-                  Create
-                </Button>
-              )}
-            </>
-          }
-        >
-          <span>Name</span>
-          <Input
-            placeholder={createModalPlaceHolder}
-            onChange={this.ctasChanged.bind(this)}
-          />
-        </StyledModal>
-      </div>
-    );
-  }
-}
-SqlEditor.defaultProps = defaultProps;
-SqlEditor.propTypes = propTypes;
-
-function mapStateToProps({ sqlLab }, { queryEditor }) {
-  let { latestQueryId, dbId, hideLeftBar } = queryEditor;
-  if (sqlLab.unsavedQueryEditor.id === queryEditor.id) {
-    const {
-      latestQueryId: unsavedQID,
-      dbId: unsavedDBID,
-      hideLeftBar: unsavedHideLeftBar,
-    } = sqlLab.unsavedQueryEditor;
-    latestQueryId = unsavedQID || latestQueryId;
-    dbId = unsavedDBID || dbId;
-    hideLeftBar = unsavedHideLeftBar || hideLeftBar;
-  }
-  const database = sqlLab.databases[dbId];
-  const latestQuery = sqlLab.queries[latestQueryId];
-
-  return {
-    hideLeftBar,
-    queryEditors: sqlLab.queryEditors,
-    latestQuery,
-    database,
-  };
-}
-
-function mapDispatchToProps(dispatch) {
-  return bindActionCreators(
-    {
-      addQueryEditor,
-      estimateQueryCost,
-      persistEditorHeight,
-      postStopQuery,
-      queryEditorSetAutorun,
-      queryEditorSetSql,
-      queryEditorSetAndSaveSql,
-      queryEditorSetTemplateParams,
-      runQueryFromSqlEditor,
-      runQuery,
-      saveQuery,
-      addSavedQueryToTabState,
-      scheduleQuery,
-      setActiveSouthPaneTab,
-      updateSavedQuery,
-      validateQuery,
-    },
-    dispatch,
+            )}
+          </>
+        }
+      >
+        <span>Name</span>
```

Comment:
```suggestion
        <span>{t('Name')}</span>
```
This string needs to be translated